commcare/src/main/java/org/motechproject/commcare/tasks/builder/CaseActionBuilder.java

```diff
@@ -120,14 +120,6 @@ public CaseActionBuilder(CommcareConfigService configService) {
             parameters.add(builder.createActionParameterRequest());
 
             builder = new ActionParameterRequestBuilder()
-                    .setDisplayName(DisplayNames.OWNER_ID)
-                    .setKey(EventDataKeys.OWNER_ID)
-                    .setType(UNICODE.getValue())
-                    .setRequired(true)
```

Comment:
why did you remove ownerId ? It should be still available, however not required. So try to set required to false